README.md

```diff
@@ -10,13 +10,82 @@ Requirements
 ------------
 
 * Django 1.6
-* django-localflavor
-* django-databrowse
 
 
 Installation
 ------------
 
+1. Install from GitHub.
+    ```sh
+        $ pip install git+git://github.com/unt-libraries/django-controlled-vocabularies.git
+    ```
+
+2. Add app and dependency to INSTALLED_APPS.
+    ```python
+        INSTALLED_APPS = (
+            'django_databrowse',
+            'accession',
+        )
+    ```
+
+3. Include the URLs.
+    ```python
+        urlpatterns = [
+            url(r'^admin/', include(admin.site.urls)),
+            url(r'^tfc/', include('accession.urls')),
+        ]
+    ```
+
+4. Sync the database and create a superuser account.
+    ```sh
+        $ python manage.py syncdb
+    ```
+
+
+Developing
+----------
+
+1. [Install Docker](http://docs.docker.com/installation/)
+
+2. Install Docker Compose
+    ```sh
+        $ pip install docker-compose
+    ```
+
+3. Clone the repository.
+    ```sh
+        $ git clone https://github.com/unt-libraries/django-controlled-vocabularies
+    ```
+
+4. Build the images.
```

Comment:
Can we eliminate this step? The build will happen automatically on `docker-compose up` if the image doesn't exist yet.